src/main/java/com/google/zxing/integration/android/IntentResult.java

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Ok. using gradle dependency now. but using the ZXing app was suggested will by Juca. Will try with  GPS barcode scanner too